numpy/lib/tests/test_io.py

```diff
@@ -303,6 +303,22 @@ def test_closing_zipfile_after_load(self):
             data.close()
             assert_(fp.closed)
 
+    @np.testing.dec.skipif(sys.version_info < (3, 6), "Works only in Python 3.6+")
+    def test_savez_bizarre_names(self):
+        keys = ['/test', '\\test', 'c:test',
+                 'test//ing', 'test/./ing', 'test/../ing',
+                 'test.npy', '', '\xe0', '\xe3', chr(0x03b1)]
```

Comment:
Can you add `'\0testing` in here too? What about `nul` or some other reserved windows name?

Reply:
Unfortunately filenames are truncated on '\0'. And backslashes are replaced with slashes on Windows. Hence not all names are preserved.

Reply:
Then I'm not sure this is something we should be testing for. Let's just document that the keys correspond to filenames, and leave it at that.

Reply:
Agree.